docs/interactions/Application_Commands.md

```diff
@@ -122,39 +169,208 @@ headers = {
 r = requests.post(url, headers=headers, json=json)
 ```
 
-This command will be available on _all_ your app's guilds.
+### Making a Guild Command
 
-> warn
-> Global commands are cached for **1 hour**. That means that new global commands will fan out slowly across all guilds, and will be guaranteed to be updated in an hour.
+Guild commands are available only within the guild specified on creation. Guild commands update **instantly**. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.
 
 To make a **guild** Slash Command, make a similar HTTP POST call, but scope it to a specific `guild_id`:
 
 ```py
 import requests
 
 
-url = "https://discord.com/api/v8/applications/<my_application_id>/guilds/<guild_id>/commands"
+url = "https://discord.com/api/v8/applications/<my_application_id>/commands"
```

Comment:
I assume this was meant to be an example for registering a user command that accidentally ended up in the guild scoped registration section?

Reply:
The header above suggests its meant to be a guild command 👀